mkosi/initrd.py

```diff
@@ -289,6 +289,12 @@ def main() -> None:
             show_docs("mkosi-initrd", DocFormat.all(), resources=r)
         return
 
+    modulesd = Path("/usr/lib/modules") / args.kernel_version
+    if not modulesd.is_dir() or not (
+        (modulesd / "modules.dep").exists() or (modulesd / "modules.dep.bin").exists()
+    ):
```

Comment:
Maybe factor this into a separate function `is_valid_modulesd` (maybe in `kmod`?), since it's used in three places in two files.

Reply:
Right, changed.